software/controller/lib/core/alarm.h

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2020, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef ALARM_H
+#define ALARM_H
+
+#include <stdint.h>
+
+enum alarm_proirity {
```

Comment:
This should be `enum class`
https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum
https://en.cppreference.com/w/cpp/language/enum

And should follow the Google code style guidelines for naming:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
modified all enums to "enum class" in the latest code